site.yml

```diff
@@ -79,3 +79,26 @@
   - { role: docker, etcd_client_port1: 2379 }
   - { role: swarm, run_as: master }
   - { role: contiv_network, run_as: master }
+
+# Netplugin master
```

Comment:
yeah, I undersatnd the swarm issue. I think this change itself will not solve the swarm issue as it will happen again once we have more than one node designated as master. I think one of the ways to solve the swarm issue is to not depend on the `run_as` variable and instead use something else like a swarm bootstrap variable/scenario (like we do in etcd). WDYT? let me try and see if I can come up with a gist which might make it a bit more clear.